src/widgets-service.js

```diff
@@ -163,7 +163,7 @@ WidgetsService.prototype.createTimeSeriesModel = function (attrs, layer, state,
   attrs.column_type = attrs.column_type || 'date';
   var dataviewModel = this._dataviews.createHistogramModel(layer, attrs);
 
-  var attrsNames = ['id', 'style', 'normalized'];
+  var attrsNames = ['id', 'style', 'title', 'normalized'];
```

Comment:
IMO if this is a "constant", we should treat it following the conventions for that, this is uppercased.